builder/amazon/chroot/builder.go

```diff
@@ -369,7 +369,7 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 	state.Put("hook", hook)
 	state.Put("ui", ui)
 	state.Put("wrappedCommand", common.CommandWrapper(wrappedCommand))
-	generatedData := &builder.GeneratedData{State: state}
+	generatedData := &gendata.GeneratedData{State: state}
```

Comment:
I think it'd be better to be more precise about this new package's name; `github.com/hashicorp/packer/common/packerbuilderdata` and also maybe rename the struct:

To call it this way:
```suggestion
	generatedData := &builderdata.State{State: state}
```

Mainly because it'd be more precise/easy to understand what this is from, from reading the pkg name.

Reply:
I think you're probably right on this one. I can do that.